Lib9c/BlockPolicy.cs

```diff
@@ -60,8 +66,6 @@ public static IBlockPolicy<PolymorphicAction<ActionBase>> GetPolicy(int minimumD
 #endif
         }
 
-        public static ActionRenderer GetRenderer() => ActionRenderer;
```

Comment:
It might be better to have a method to get two kinds of renderers like:

```suggestion
        public static IEnumerable<IRenderer<ActionBase>> GetRenderers() => new[] { BlockRenderer, ActionRenderer };
```

… so that client code can pass it to the `BlockChain<ActionBase>()` constructor:

```csharp
new BlockChain<ActionBase>(…, renderers: BlockPolicy.GetRenderers());
```